quint/test/repl.test.ts

```diff
@@ -143,7 +143,7 @@ describe('repl ok', () => {
     )
     const output = dedent(
       `>>> 1 + false
-      |static analysis error: <input>:1:1 - error: Couldn't unify int and bool
+      |static analysis error: <input>:1:1 - error: QNT000: Couldn't unify int and bool
```

Comment:
We have a lot of colons piling up here, might something like be more standard/reabable:

```
      |static analysis error: <input>:1:1 - error [QNT000]: Couldn't unify int and bool
```

I'm not sure here, and this is low stakes. Just a thought.

Reply:
I did this instead:
```
      |static analysis error: <input>:1:1 - error: [QNT000] Couldn't unify int and bool
```